src/CommandingOfficers/Tech.java

```diff
@@ -129,6 +138,12 @@ public OverchargeAbility(Commander commander, String abilityName, double ability
       damageBuff = new CODamageModifier(buff);
       defenseBuff = new CODefenseModifier(buff);
       healAmount = healAmt;
+
+      // Only mechanical units get the firepower boost.
+      long exclude = UnitModel.TROOP;
+      ArrayList<UnitModel> models = commander.getAllModels(Tech.mechanicalUnits, true, exclude);
+      for( UnitModel m : models )
+        damageBuff.addApplicableUnitModel(m);
```

Comment:
Poor Bikes. What did they ever do to you?

Reply:
We don't have a `Mechanical` flag, or maybe they'd qualify (though for balance they probably shouldn't, not that "balance" is really part of the picture here). Sorry pals.